src/Microsoft.Health.Dicom.Core/Features/Query/QueryResource.cs

```diff
@@ -17,5 +17,6 @@ public enum QueryResource
         AllInstances,
         StudyInstances,
         StudySeriesInstances,
+        WorkitemInstances,
```

Comment:
Does this fit here? I am just thinking ResourceType { DicomInstance, WorkItem}

Reply:
Removed